dvc/scm/base.py

```diff
@@ -165,3 +166,15 @@ def belongs_to_scm(self, path):
 
     def close(self):
         """ Method to close the files """
+
+    def brancher(
+        self,
+        root_dir,
+        revs=None,
+        all_branches=False,
+        all_tags=False,
+        all_commits=False,
+    ):  # pylint: disable=no-self-use, unused-argument
+        yield "workspace", LocalTree(
```

Comment:
Do we need to return workspace from here? It looks like it makes mess, i.e. we need to skip it later.

Reply:
Also `LocalTree` has absolutely nothing to do with Base SCM